### ISSUE TITLE
nunjucks escapes & characters

### ISSUE THREAD

**Issue by philippevk** [eykrehbein/strest#135]
**Describe the bug**
Nunjucks escapes ampersand symbols by default, which makes it impossible to have query string elements in a rendered string (https://mozilla.github.io/nunjucks/api.html#autoescaping)

**To Reproduce**
- Have request1 calling an endpoint that returns an url with a 2+ queryString items
- Have request2 use `url: "<$ previous.content $>"`
- Notice that "&" is escaped to "&amp;amp;"

**Expected behavior**
- queryString is passed as-is in the url

you can just pass `autoescape: false` to nunjucks.configure to disable this

**Comment by jgroom33** [comment]
Thanks for identifying this.

This test is being added:

```yaml
version: 2

requests:
  setParam:
    request:
      url: https://postman-echo.com/post
      method: POST
      postData:
        mimeType: application/json
        text:
          param: https://jsonplaceholder.typicode.com/posts?userId=1&id=3
    validate:
    - jsonpath: status
      expect: 200
  useParam:
    request:
      url: <$ setParam.content.data.param $>
      method: GET
    validate:
    - jsonpath: content.0.id
      expect: 3
```

**Comment by jgroom33** [comment]
Resolved with release 2.4.1
Please confirm

**Comment by philippevk** [comment]
confirmed, thanks (and sorry for the long delay)
